### PR TITLE
Finish GET fixtures

### PR DESCRIPTION
**Pull request by msecret**
This adds the rest of the fixtures for getting data, doesn't add routes for when data
is modified.

### CODE REVIEW

**Comment by jeremiak** [comment]
This is totally cool, and adds a whole bunch of data to the app!

At this point all of the views should be visible to the user, but they still won't be able to do any write actions such as creating apps, service instances, etc.

File: test/api.js [hunk 1]
```diff
@@ -42,7 +42,11 @@ module.exports = function api(smocks) {
     label: 'App stats',
     path: `${BASE_URL}/apps/{guid}/stats`,
     handler: function (req, reply) {
-      reply(appStats);
+      var app = apps.filter((a) => a.guid === req.params.guid).pop();
+      var response = Object.assign({}, appStats, {
+        state: app.state
```
Comment (msecret) [inline test/api.js]: Should we also set current mem and disk to 0 or null so it's a more correct response?

Comment (msecret) [inline test/api.js]: Wait, doesn't `stats` not return anything for stopped apps?

Comment (jeremiak) [inline test/api.js]: Yup, you're right. Looks like it returns the following `400` error:

```
{
  "code": 200003,
  "description": "Could not fetch stats for stopped app: help",
  "error_code": "CF-AppStoppedStatsError"
}
```

I'll fix that, nice catch

**Pull request merged**